src/components/Link/Link.tsx

```diff
@@ -52,30 +26,7 @@ export const Link = forwardRef(
     { children, className, to, ...props }: LinkProps,
     ref: Ref<HTMLAnchorElement>
   ) => {
-    const [isExternal, setExternal] = useState(false);
-    const isFile = isFileLink(to);
-
-    // Ensures 'window' is present
-    useEffect(() => {
-      if (to) {
-        setExternal(isExternalLink(to));
-      }
-    }, []);
-
-    return isExternal || isFile ? (
-      <a
-        className={className}
-        download={isFile}
-        href={to}
-        ref={ref}
-        rel="noopener noreferrer"
-        target="_blank"
-        {...props}
-      >
-        {children}
-        {isExternal && !isFile && <ExternalLinkMarker />}
-      </a>
-    ) : (
+    return (
       <RouterLinkWrapper href={to} ref={ref}>
         <a className={className} href={to} {...props}>
```

Comment:
Is it worth keeping `download={isFile}`?

Reply:
yes, that's a good point. I will update.